docs/reference/technology/connectivity/networking.md

```diff
@@ -0,0 +1,93 @@
+### IP networking
+
+IP Networking in Mbed OS is layered in three clearly defined API levels. The diagram below shows the layers next to the closest matching [OSI model](https://en.wikipedia.org/wiki/OSI_model) layers.
```

Comment:
More credible?
This is first hit in the Google if you search for OSI model. So it is the most linked one.
I'm not sure there are more stable sites that we can confidently link on, and still assume after couple of years that the link is still valid.

OSI Model is a theoretical model from academic text book. Defined by the ISO organization. Most computer science students recognize it. But because it is theoretical, there are not many sites that host proper textbook explanation about it, others that Wikipedia. If you can recommend me one, I can change the link.

Proper standard about it, is behind pay-wall https://ieeexplore.ieee.org/document/1094702/